packages/launch/src/focused-launch/index.tsx

```diff
@@ -67,28 +73,39 @@ const FocusedLaunch: React.FunctionComponent< Props > = ( { siteId } ) => {
 			<div className="focused-launch__step">
 				<div className="focused-launch__data-input">
 					<div className="focused-launch__section">
-						<DomainPicker
-							header={
-								<>
-									<label className="focused-launch__label">
-										{ __( '2. Confirm your domain', __i18n_text_domain__ ) }
-									</label>
-									<p className="focused-launch__mobile-commentary focused-launch__mobile-only">
-										<Icon icon={ bulb } /> 46.9% of globally registered domains are .com
-									</p>
-								</>
-							}
-							existingSubdomain={ siteDomainName }
-							currentDomain={ siteDomainName }
-							onDomainSelect={ noop }
-							initialDomainSearch={ domainSearch }
-							showSearchField={ false }
-							analyticsFlowId="focused-launch"
-							analyticsUiAlgo="focused_launch_domain_picker"
-							onDomainSearchBlur={ () => noop( 'TODO: on domain search blur' ) }
-							onSetDomainSearch={ () => noop( 'TODO: on set domain search' ) }
-							quantity={ 3 }
-						/>
+						{ hasPaidDomain ? (
+							<>
+								<label className="focused-launch__label">
+									{ __( 'Your domain', __i18n_text_domain__ ) }
+								</label>
+								<LockedPurchasedItem domainName={ sitePrimaryDomain?.domain || '' } />
+							</>
+						) : (
+							<DomainPicker
+								header={
+									<>
+										<label className="focused-launch__label">
+											{ stepNumber++ }. { __( 'Confirm your domain', __i18n_text_domain__ ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( '2. Confirm your domain' )` *ES Score: 9*